chrome/nav.js

```diff
@@ -972,7 +963,7 @@ function displayStimTimer ( ukey, usebtn, data ) {
 	}
 	else {
 		// We have data, display current addiction
-		timerDiv.appendChild( doc.createTextNode('Stims used:') );
+		timerDiv.appendChild( doc.createTextNode('Stim used:') );
```

Comment:
Is it?
I changed "Adds" for "Add" since you suggested it and I agree it is.
But Stim is short for Stimulant. So Stim Chip is Stimulant Chip.
You could have used 1 Stim or many Stims, so the grammar would be:
"You are clear of the stimulant effect of the Stim Chip" or "You are clear of the Stimulant's effect"...

Reply:
AFAIK the common lingo for stimulant chips is ... stims.